builtin/game/chat.lua

```diff
@@ -500,8 +506,8 @@ core.register_chatcommand("teleport", {
 		end
 
 		local has_bring_priv = core.check_player_privs(name, {bring=true})
-		local missing_bring_msg = "You don't have permission to teleport " ..
-			"other players (missing bring privilege)"
+		local missing_bring_msg = S("You don't have permission to teleport " ..
+			"other players (missing privilege: bring).")
```

Comment:
Translators might accidentally translate the privilege name as well. `@1` would be totally save.